ui/pages/settings/flask/snaps-list-tab/index.scss

```diff
@@ -0,0 +1,28 @@
+.snaps-list-wrapper {
```

Comment:
Nit: These class names do not comply with the naming convention this project uses, which is BEM (Block, Element, Modifier). See here for more information: http://getbem.com/

Typically the component is a "block", and each piece inside the component is an "element". So we would have a top-level class for the block, and sub classes within that block called `&__[element-name]`.

Reply:
Ah! I'll make those changes.